ansible/vars/topo_wan-4link.yml

```diff
@@ -0,0 +1,102 @@
+topology:
+  topo_type: wan
+  VMs:
+    ARISTA01T1:
+      vlans:
+        - 0
+        - 1
+        - 2
+        - 3
+      vm_offset: 0
+    ARISTA02T1:
+      vlans:
+        - 10
+        - 11
+        - 12
+        - 13
+      vm_offset: 1
+  DUT:
+    loopback:
+      ipv4:
+        - 10.1.0.32/32
+      ipv6:
+        - FC00:1::32/128
+wan_dut_configuration:
+  vlab-01:
+    interfaces:
+      PortChannel101:
+        intfs: [0, 1, 2, 3]
+        ipv4: 10.0.0.56/31
+        ipv6: FC00::71/126
+      PortChannel102:
+        intfs: [10, 11, 12, 13]
+        ipv4: 10.0.0.58/31
+        ipv6: FC00::75/126
+configuration_properties:
+  common:
+    dut_asn: 65100
+    dut_type: InternetBackboneRouter
+    swrole: leaf
+configuration:
+  ARISTA01T1:
+    properties:
+    - common
+    bgp:
+      asn: 64600
+      peers:
+        65100:
+        - 10.0.0.56
+        - FC00::71
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.29/32
+        ipv6: 2064:100::1d/128
+      Ethernet1:
+        dut_index: 0
+        lacp: 1
+      Ethernet2:
+        dut_index: 0
+        lacp: 1
+      Ethernet3:
+        dut_index: 0
+        lacp: 1
+      Ethernet4:
+        dut_index: 0
+        lacp: 1
+      Port-Channel1:
+        ipv4: 10.0.0.57/31
+        ipv6: fc00::72/126
+    bp_interface:
+      ipv4: 10.10.246.29/24
+      ipv6: fc0a::1d/64
+  ARISTA02T1:
+    properties:
+    - common
+    bgp:
+      asn: 64600
+      peers:
+        65100:
+        - 10.0.0.58
+        - FC00::75
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.30/32
+        ipv6: 2064:100::1e/128
+      Ethernet1:
+        dut_index: 0
+        lacp: 1
+      Ethernet2:
+        dut_index: 0
+        lacp: 1
+      Ethernet3:
+        dut_index: 0
+        lacp: 1
+      Ethernet4:
+        dut_index: 0
```

Comment:
If only 1 DUT is involved in this topology, why bother adding the `dut_index` parameter?

Reply:
remove dut_index as there is only one dut in this topo and the default index is 0.